reporting_app/__init__.py

```diff
@@ -303,43 +303,91 @@ def report_samples(view_type):
     )
 
 
-@app.route('/projects/<project_id>')
+@app.route('/projects/<project_ids>')
 @flask_login.login_required
-def report_project(project_id):
+def report_project(project_ids):
+    if project_ids == 'SGP':
+        id_list = []
+        for p in rest_api().get_documents('lims/project_info', match={'project_status': 'all'}):
```

Comment:
Can we change the `project_status` to `'open'` to avoid old projects starting with S.